docs/UserGuide.md

```diff
@@ -417,8 +490,8 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 * By not entering any prefixes, it will sort by name in ascending order.
 * Enter the right amount of prefixes! For example: if you only sort by status, use only `s/`. 
```

Comment:
I think `if you only sort by status, use only s/` can be replaced with `use only s/ if you are sorting by status.`